Manager/PasteboardItem.h

```diff
@@ -1,15 +1,15 @@
 #import <UIKit/UIKit.h>
 
 @interface PasteboardItem : NSObject
-@property(atomic, assign)NSString* bundleIdentifier;
-@property(atomic, assign)NSString* displayName;
-@property(atomic, assign)NSString* content;
-@property(atomic, assign)NSString* imageName;
-@property(atomic, assign)BOOL hasPlainText;
-@property(atomic, assign)BOOL hasLink;
-@property(atomic, assign)BOOL hasMusicLink;
-@property(atomic, assign)BOOL hasColor;
-@property(atomic, assign)BOOL hasImage;
+@property (nonatomic, copy) NSString* bundleIdentifier;
```

Comment:
## Objective-C
- It's safe and more efficient to use `nonatomic` keyword for all objc properties because all of your codes will be executed on main queue.
- When ARC is enabled: use `weak/strong` keywords instead of `assign/retain`.
- It's a common practice to use `copy` keyword for `NSString`s.

Reply:
- I've heard too much, yet too little info about atomic and nonatomic, so what's the point in changing it other than "speed"?

- Why use weak/strong? I actually don't know the difference.

- Same question here, why use copy?

Reply:
[Objective-C Memory Management with ARC - raywenderlich.com](https://www.youtube.com/watch?v=EKf3t1ytZZs)

Well, I'm not a professor. Tell me if I'm wrong here.